src/import/metrics.rs

```diff
@@ -45,4 +45,46 @@ lazy_static! {
         exponential_buckets(0.001, 2.0, 20).unwrap()
     )
     .unwrap();
+    pub static ref IMPORT_RANGE_DELIVERY_DURATION: Histogram = register_histogram!(
+        "tikv_import_range_delivery_duration",
+        "Bucketed histogram of import range delivery duration",
+        exponential_buckets(0.001, 2.0, 20).unwrap()
+    )
+    .unwrap();
+    pub static ref IMPORT_SPLIT_SST_DURATION: Histogram = register_histogram!(
+        "tikv_import_split_sst_duration",
+        "Bucketed histogram of import split sst duration",
+        exponential_buckets(0.1, 2.0, 20).unwrap()
```

Comment:
I think you should change this base value and others below too.

Reply:
This base value should keep 0.1 because split sst will take a long time.